Extensions/xkit_patches.js

```diff
@@ -201,6 +232,13 @@ XKit.tools.make_gist = function(text, name) {
 	});
 };
 
+/**
+ * Injects a function into the DOM scope.
```

Comment:
Although `addt` is not actually passed as a parameter. Will update with parts of your eloquence.